app/workers/repo_clone_worker.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+require 'open3'
+require 'shellwords'
+
+class RepoCloneWorker
+  include Sidekiq::Worker
+  sidekiq_options queue: :event
+
+  def perform(mission_params)
```

Comment:
**[rubocop]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
Metrics/AbcSize: Assignment Branch Condition size for perform is too high. [16.09/15]